src/retry.ts

```diff
@@ -89,6 +89,12 @@ export async function getRetryConfig(err: GaxiosError) {
 function shouldRetryRequest(err: GaxiosError) {
   const config = getConfig(err);
 
+  // If the user has explicitly aborted the request, don't retry.
+  // "type" is populated on the error objects created by node-fetch.
+  if (err.type === 'aborted') {
```

Comment:
Makes me wonder - should we type the `type` above as `type?: 'ABORTED';`?

Reply:
opted for an alternate approach in which we use `err.name`.